### ISSUE TITLE
`admin.cy.js` test is failing due to a slow performance query

### ISSUE THREAD

**Issue by pdcp1** [responsible-ai-collaborative/aiid#2272]
Staging deploys are failing due to a bad performance on the `FindUsers` graphql query. It takes ~1 minute to complete and the test fails before that.

```
query FindUsers {
    users {
      roles
      userId
      first_name
      last_name
      adminData {
        email
        disabled
        creationDate
        lastAuthenticationDate
      }
    }
  }
```

A temporary solution to unblock the deployment process is to skip this test. In the meantime, we must find a better solution to that [Users table](https://staging-aiid.netlify.app/admin/). A pagination approach could help.

**Comment by kepae** [comment]
Let's skip the test temporarily to unblock out deploys and keep this issue open to track it.

**Comment by cesarvarela** [comment]
The reason for this is that the atlas admin API is extremely slow:

https://github.com/responsible-ai-collaborative/aiid/blob/master/site/realm/functions/apiRequest.js#L28-L35

One possible solution is to remove the `adminData` tree from the main query and fetch each user's `adminData` individually.

**Comment by pdcp1** [comment]
Closing this since it was already merged into Staging
